### PR TITLE
Multiple images support (task #3945)

### PR DESCRIPTION
**Pull request by georgeconstantinou**
- Added lightbox functionality in image galleries (requires [PR](https://github.com/QoboLtd/cakephp-utils/pull/41) merge and release).
- Added insert gallery button and custom functionality:
insert-gallery

### CODE REVIEW

File: webroot/js/tinymce.init.js [hunk 1]
```diff
@@ -9,6 +9,16 @@
         }
 
         config.file_browser_callback = elFinderBrowser;
+        config.setup = function (editor) {
+            editor.addButton('gallery', {
+                tooltip: 'Insert gallery',
+                icon: 'image',
```
Comment (mamchenkov) [inline webroot/js/tinymce.init.js]: @georgeconstantinou maybe switch to 'camera' just to have a different icon

Comment (georgeconstantinou) [inline webroot/js/tinymce.init.js]: @mamchenkov supported icons are listed here: https://github.com/tinymce/tinymce/blob/master/src/skins/lightgray/src/main/less/Icons.less

Comment (mamchenkov) [inline webroot/js/tinymce.init.js]: I was looking at font awesome ... this ones are not very visual :)

Comment (georgeconstantinou) [inline webroot/js/tinymce.init.js]: :smile:

**Pull request merged**